modules/javafx.graphics/src/main/native-prism-d3d/D3DMeshView.h

```diff
@@ -50,15 +50,15 @@ class D3DMeshView {
     void render();
 
 private:
-    D3DContext *context;
-    D3DMesh *mesh;
-    D3DPhongMaterial *material;
+    D3DContext *context = NULL;
+    D3DMesh *mesh = NULL;
+    D3DPhongMaterial *material = NULL;
     D3DLight lights[MAX_NUM_LIGHTS];
-    float ambientLightColor[3];
-    int numLights;
-    bool lightsDirty;
-    int cullMode;
-    bool wireframe;
+    float ambientLightColor[3] = {0};
+    int numLights = 0;
+    bool lightsDirty = TRUE;
+    int cullMode = D3DCULL_NONE;
+    bool wireframe = FALSE;
```

Comment:
It seems like you're using `false` and `FALSE` interchangably (see `D3DPhongMaterial.h` L58). I would suggest using the `false` keyword with the builtin type `bool`, and the `FALSE` constant with the Win32 type `BOOL`.

Reply:
The original code used the Win-only type. I thought it should be changed too. I should probably do that,